src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -177,6 +186,48 @@ protected int submit(List<O> ops) throws InterruptRunException, IllegalStateExce
         return submit(ops, 0, ops.size());
     }
 
+    
+	private boolean invokePathRead(final PathOperation<? extends PathItem> pathOp) {
+		final int READER_TIMEOUT_MS = 100;
```

Comment:
1. Why not 1 instead of 100?
2. Why not to place in some kind of interface like "PravegaConstants"?

Reply:
1. because TIMEOUT has some sort of a bug. And its value must be big enough to start reading the stream. It contradicts with docs, but i couldn't get consistent results with 1.
2. I've moved it to PravegaConstants.java file. Should it be an interface as you said?

Reply:
There are constants interface already existing:
https://github.com/emc-mongoose/mongoose-storage-driver-pravega/blob/master/src/main/java/com/emc/mongoose/storage/driver/pravega/Constants.java
Please update your branch first